src/commands.rs

```diff
@@ -53,6 +53,16 @@ fn get_port() -> u16 {
         .unwrap_or(DEFAULT_PORT)
 }
 
+/// Check if ignoring all response errors
+fn ignore_all_server_io_errors() -> bool {
+    let ignore;
+    match env::var("SCCACHE_IGNORE_SERVER_IO_ERROR") {
+        Ok(val) => ignore = val,
+        Err(_e) => ignore = "none".to_string(),
+    }
+    ignore == "1"
```

Comment:
```suggestion
    match env::var("SCCACHE_IGNORE_SERVER_IO_ERROR") {
        Ok(ignore_server_error) => ignore_server_error == "1"
        Err(_) => false
    }
```
This should be simplify-able a bit :)